tools/codegen/pkg/interfacegen/generator.go

```diff
@@ -17,13 +17,16 @@ package interfacegen
 import (
 	"bytes"
 	"fmt"
+	"go/format"
 	"io/ioutil"
 	"os"
 	"text/template"
 
-	"github.com/golang/protobuf/proto"
-	"github.com/golang/protobuf/protoc-gen-go/descriptor"
+	"github.com/gogo/protobuf/proto"
+	"github.com/gogo/protobuf/protoc-gen-gogo/descriptor"
```

Comment:
here: because we use gogo to compile the other protos. having both gogo and golang protobuf deps causes all sorts of issues. I think we want everything to be gogo at the moment.